README.md

```diff
@@ -0,0 +1,138 @@
+| [`incident`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.incident) | Model for incidents in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+| [`opportunity`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.opportunity) | Model for opportunities in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+| [`phonecall`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.phonecall) | Model for phone calls in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+| [`systemuser`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.systemuser) | Model for system users in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+
+### Materialized Models
+Each Quickstart transformation job run materializes 7 models if all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as `view`, `table`, or `incremental`.
+
+<!--section-end-->
+
+## How do I use the dbt package?
+
+### Step 1: Prerequisites
+To use this dbt package, you must have the following:
+
+- At least one Fivetran Microsoft Dynamics 365 CRM connection syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.
+
+#### Databricks dispatch configuration
+If you are using a Databricks destination with this package, you must add the following (or a variation of the following) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
+
+### Step 2: Install the package
+Include the following Microsoft Dynamics 365 CRM package version in your `packages.yml` file:
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+```yml
+packages:
+  - package: fivetran/dynamics_365_crm
+    version: [">=0.1.0", "<0.2.0"] # we recommend using ranges to capture non-breaking changes automatically
+```
+
+### Step 3: Define database and schema variables
+
+By default, this package runs using your destination and the `dynamics_365_crm` schema. If this is not where your Microsoft Dynamics 365 CRM data is (for example, if your Microsoft Dynamics 365 CRM schema is named `dynamics_365_crm_fivetran`), add the following configuration to your root `dbt_project.yml` file:
```

Comment:
Do you know if `dynamics_365_crm` is the default schema name? I went to create a new connection and saw the default was actually `dynamics_365`. While it doesn't support our naming convention, we should (and I believe are required with Quickstart) for the default schema name to be the same default name provided by the connector.

Reply:
It actually looks like the default schema used in the package is [dynamics_365](https://github.com/fivetran/dbt_dynamics_365_crm/blob/6c8e0f0cb4078006fdf6b4b8308d177fe565a7ad/models/src_dynamics_365_crm.yml#L6). Let's update this to match the default in the package and connector.